packages/cli/src/bin/pipcook-job.ts

```diff
@@ -11,7 +11,8 @@ import { tunaMirrorURI } from '../config';
 const PipelineStatus = [ 'creating', 'running', 'success', 'fail' ];
 
 async function list(): Promise<void> {
-  const jobs = await get(`${route.job}/list`);
+  let jobs = await get(`${route.job}/list`);
+  jobs = jobs.rows;
   console.table(jobs.map((item: any) => {
```

Comment:
Using `jobs.rows` here to keep the `jobs` constant.

Reply:
fixed